Doc/reference/expressions.rst

```diff
@@ -582,6 +582,11 @@ is already executing raises a :exc:`ValueError` exception.
    :attr:`~BaseException.__traceback__` attribute stored in *value* may
    be cleared.
 
+   .. versionchanged:: 3.12
+
+      The second signature \(type\[, value\[, traceback\]\]\) is deprecated now.
+      It is still available, but may be removed in a future version of Python.
```

Comment:
```suggestion
      The second signature \(type\[, value\[, traceback\]\]\) is deprecated and
      may be removed in a future version of Python.
```

Reply:
There are a few other places in the docs (I used index search for throw):

https://docs.python.org/3.12/reference/datamodel.html?highlight=throw#coroutine.throw
https://docs.python.org/3.12/reference/expressions.html?highlight=throw#agen.athrow

And as @kumaraditya303 mentioned, we need an entry in 

https://github.com/python/cpython/blob/main/Doc/whatsnew/3.12.rst